plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/standard/StandardDeployPreferencesConverter.java

```diff
@@ -0,0 +1,35 @@
+package com.google.cloud.tools.eclipse.appengine.deploy.standard;
+
+import com.google.cloud.tools.appengine.api.deploy.DefaultDeployConfiguration;
+
+public class StandardDeployPreferencesConverter {
+
+  private StandardDeployPreferences preferences;
+
+  public StandardDeployPreferencesConverter(StandardDeployPreferences preferences) {
+    this.preferences = preferences;
+  }
+
+  public DefaultDeployConfiguration toDeployConfiguration() {
```

Comment:
Would it make sense to call this an `Applicator` and have it `applyTo()` a deploy configuration?